Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM node:10-alpine
+
+RUN apk update && apk upgrade && \
+    apk add --no-cache yarn
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+ENV HOST 0.0.0.0
+
+ARG BRANCH
+ENV NODE_ENV $BRANCH
+
+RUN cd /usr/src/app/
+COPY . .
+RUN yarn
+RUN cd components/AddToCalendar/src && npm install --production
+RUN cd components/Calender && npm install --production
+RUN cd components/conference-cal && npm install --production
+
```

Comment:
why is this a seprate install, should it not be part of package.json

Reply:
they are stand-alone, packages/components.  There are many ways to deal with these.  Design systems, mono repos, learna, yarn workspaces.  We need to decide.  For now they can sit here.